.devcontainer/devcontainer.json

```diff
@@ -1,11 +0,0 @@
-{
-    "name": "Playwright",
-    "image": "mcr.microsoft.com/playwright:next",
-    "postCreateCommand": "npm install && npm run build && apt-get update && apt-get install -y software-properties-common && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && add-apt-repository \"deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable\" && apt-get install -y docker-ce-cli",
```

Comment:
Please don't delete this file, we (lusha or me) use it with GitHub codespaces.

Reply:
oops, this sneaked in after me fighting with misbehaving code formatting :)